recipes/pantools/meta.yaml

```diff
@@ -32,13 +32,13 @@ requirements:
     - blast
     - mash =2.3
     - fastani
-      #- busco =5  #causes conflicts on macOS
+    - busco =5  # [linux]
```

Comment:
What kind of conflicts does it cause? Isn't busco an essential dependency?

Reply:
BUSCO is essential only for one subcommand of PanTools luckily which likely no one runs on OSX systems, but it seems like the conflict is related to openMP differences between Linux and OSX (this causes conflicts between BUSCO, fastani, kmc and python). However, I am not familiar enough with openMP myself to fix this.